apps/documentation/src/components/MdxProvider/MdxProvider.tsx

```diff
@@ -36,6 +38,8 @@ export default function MdxProvider({
         tr: MdxTable.Tr,
         th: MdxTable.Th,
         td: MdxTable.Td,
+        p: MdxText,
```

Comment:
`MdxText` is only used as component for paragraph so it should be named `MdxParagraph`